Dima/netflix-roulette/src/Components/Films/FilmCard.tsx

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
 import IFilm from '../../interfaces/IFilm'
 
+import defaultImg from '../../assets/default.jpg'
+
+
 
 const FilmCard = ({ film }: { film: IFilm }) => {
+
+  // const imageStyle: any = {
+  //   backgroundImage: `url(${film.poster_path}), url(${defaultImg});`
+  // }
+
+  function handleImgError (e: React.SyntheticEvent<HTMLImageElement, Event>) {
+    const target = e.target as HTMLImageElement
+    target.src = defaultImg
+  } 
+
   return (
     <div className='film__card'>
-      <img src = { film.poster_path } alt = { film.title } />
+      <img 
+        // className = { imageStyle }
+        onError = { handleImgError }
+        src={ film.poster_path } 
+        alt={ film.title }
+      />
```

Comment:
:optional 
I see that you have an image only inside film card, but it would be grate if you've made an Image component that has this onError handler, so you wouldn't need to duplicate this logic, if you had to show placeholder on another page.